tests/ui/extern-flag/redundant-force-extern.rs

```diff
@@ -0,0 +1,11 @@
+// check-pass
+// aux-crate:force:panic_handler=panic_handler.rs
+// compile-flags: -Zunstable-options --crate-type dylib
+// edition:2018
+
+#![no_std]
+
+extern crate panic_handler;
+
+fn main() {
```

Comment:
```suggestion
fn main() {}
```
Nit: rustfmt is not run on tests automatically because it breaks expected diagnostic annotations, but it's generally better to keep tests formatted too.